<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="GameWidget" name="frameJeu">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>1150</width>
     <height>850</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="display_image">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>340</width>
     <height>255</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonInitWebCam">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Initialiser Webcam</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GameWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>gamewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
